src/drawer.stories.ts

```diff
@@ -49,11 +49,7 @@ const meta: Meta = {
     button?.addEventListener('click', () => {
       if (drawer) {
         drawer.open = !drawer.open;
-      }
-    });
 
-    const observer = new MutationObserver(() => {
-      if (drawer instanceof GlideCoreDrawer) {
         addons.getChannel().emit(UPDATE_STORY_ARGS, {
```

Comment:
It took me a minute to figure out why my `--width` on the host in [this](https://github.com/CrowdStrike/glide-core/pull/745/files#diff-3c30b204b676b2b96384fe53d8f5f230b3696f7b726c0715252127fea3d9f3e5R60) test was getting wiped out before the screenshot was taken.

Everything not included in `updatedArgs` is reverted back to its original `args` value, and `--width` starts off as an empty string. So I simply got rid of the mutation observer and am now emitting this event on "click" above.